backend/src/main/java/gov/cdc/usds/simplereport/api/pxp/PatientExperienceController.java

```diff
@@ -36,12 +38,15 @@
 import org.springframework.web.bind.annotation.RestController;
 
 /**
- * Note that this controller self-authorizes by means of this @PreAuthorize annotation and its
- * routes are set to permitAll() in SecurityConfiguration. If this changes, please update the
- * documentation on both sides
+ * Note that this controller self-authorizes by means of this {@link PreAuthorize} annotation and
+ * its routes are set to permitAll() in SecurityConfiguration. If this changes, please update the
+ * documentation on both sides.
+ *
+ * <p>Similarly, this controller sends information to the audit log via a {@link PostAuthorize}
  */
 @PreAuthorize(
     "@patientLinkService.verifyPatientLink(#body.getPatientLinkId(), #body.getDateOfBirth())")
+@PostAuthorize("@restAuditLogManager.logRestSuccess(#request, returnObject)")
```

Comment:
This requires having an HttpServletRequest argument to every method, which I don't love, but it's a lot less magical than the other options.